src/world/Game/AI/AIInterface.cpp

```diff
@@ -925,20 +925,20 @@ void AIInterface::_UpdateCombat(uint32 p_time)
 //				m_Unit->SetUInt64Value(UNIT_FIELD_TARGET, 0);
 					resetNextTarget();
 
-					WorldPacket data(SMSG_MESSAGECHAT, 100);
-					string msg = "%s attempts to run away in fear!";
-					data << (uint8)CHAT_MSG_CHANNEL;
-					data << (uint32)LANG_UNIVERSAL;
-					data << (uint32)(strlen(TO< Creature* >(m_Unit)->GetCreatureInfo()->Name) + 1);
-					data << TO< Creature* >(m_Unit)->GetCreatureInfo()->Name;
-					data << (uint64)0;
-					data << (uint32)(msg.size() + 1);
-					data << msg;
-					data << uint8(0);
+					//WorldPacket data(SMSG_MESSAGECHAT, 100);
+					char msg[256] = "%s attempts to run away in fear!";
```

Comment:
Don't do this please!
Either make it a string literal ( const char* msg = "blahblah"; ), or a C++ string ( string msg = "blahblah"; ).